tools/cd_scripts/create_binaries.sh

```diff
@@ -0,0 +1,67 @@
+#! /bin/bash
```

Comment:
NIT: rename to package_gcsfuse.sh as the script creates packages and not binaries

Reply:
done